src/dialog/overflow-menu/overflow-menu.stories.ts

```diff
@@ -40,4 +52,18 @@ storiesOf("Overflow Menu", module)
 			</ibm-overflow-menu>
 			<ibm-dialog-placeholder></ibm-dialog-placeholder>
 		`
+	}))
+	.add("Dynamic", () => ({
+		template: `
+			<span>Dynamic OverflowMenu, using the optionCount knob to change the number of menu options</span>
```

Comment:
```suggestion
			<span>
				Dynamic <code>OverflowMenu</code>, using the <code>optionCount</code> knob
				to change the number of menu options
			</span>
```
to make code read as code